python/kfserving/kfserving/api/kf_serving_client.py

```diff
@@ -27,15 +27,16 @@ class KFServingClient(object):
     '''KFServing Client Apis.'''
 
     def __init__(self, config_file=None, context=None,
-                 client_configuration=None, persist_config=True):
-        if utils.is_running_in_k8s():
-            config.load_incluster_config()
-        else:
+                 client_configuration=None, persist_config=True,
+                 force_kubeconfig=False):
+        if force_kubeconfig or utils.is_running_in_k8s() == False:
```

Comment:
nit: `force_kubeconfig or not utils.is_running_in_k8s()`

Reply:
done.